### PR TITLE
Feature 4 Changes

### PR DESCRIPTION
**Pull request by ericguo1230**

### CODE REVIEW

File: src/main/java/forms/RegisterForm.java [hunk 1]
```diff
@@ -1,4 +1,27 @@
...
+    private String password;
+    private String confirmpassword;
+
+    public RegisterForm(String title) {
+        super(title);
+    }
+
+    @Override
+    boolean validateForm() {
+        return false;
```
Comment (danielqiuu) [inline src/main/java/forms/RegisterForm.java:16]: Can you work on the validation. i.e. password and confirmpassword are the same, and that theyre 8+ characters and stuff like that.
Another thing would be email validation. i.e. contains the @

**Review by danielqiuu — CHANGES_REQUESTED** [review]
Implement validation and next time you open a pr, perhaps write a short desc of what code you wrote. (it seems like you created the form for a user to register)

File: src/main/java/forms/RegisterForm.java (hunk 1 above)
Comment (lukecheseldine) [reply src/main/java/forms/RegisterForm.java:16]: agree needs validation. no need to go crazy with it tho, doesn't have to be flawless

File: src/main/java/forms/RegisterForm.java [hunk 2]
```diff
@@ -1,4 +1,27 @@
 package forms;
 
-public class RegisterForm {
+public class RegisterForm extends Form{
+
+    private String username;
+    private String email;
+    private String password;
+    private String confirmpassword;
```
Comment (lukecheseldine) [inline src/main/java/forms/RegisterForm.java:8]:
```suggestion
    private String confirmPassword;
```

File: src/main/java/forms/RegisterForm.java [hunk 3]
```diff
@@ -1,4 +1,27 @@
...
+    boolean validateForm() {
+        return false;
+    }
+
+    @Override
+    void submitForm() {
+
+    }
+    public void createUser(String username, String email, String password){
+        //call createUser method in user class using database interactor
```
Comment (lukecheseldine) [inline src/main/java/forms/RegisterForm.java:24]:
```suggestion
        // TODO: call createUser method in user class using database interactor
```

**Review by lukecheseldine — CHANGES_REQUESTED** [review]
good start eric! like daniel said, just do the simple validation, and we'll add in the database stuff soon. remember to reformat code in intellij before committing, and also remember to request everyone to review this or they won't see it! keep it up :)

**Review by claregillis — DISMISSED** [review]
It looks like you don't have the entities package in Main, User, or View. I think you need to pull the newest changes in the main branch and merge them into yours before you can merge your branch into main. otherwise looks good!

**Review by lukecheseldine — APPROVED** [review]

**Review by danielqiuu — APPROVED** [review]

**Review by ethanrstark — APPROVED** [review]

**Review by MarkianV — APPROVED** [review]
No issues. Consider having title for register page default to 'Register".

**Pull request merged**